views/edit-student.ejs

```diff
@@ -11,6 +11,10 @@
 				<label for="last-name">Last Name</label>
 				<input type="text" class="form-control" name="last_name" value="<%=student.last_name%>" required>
 			</div>
+			<div class="form-group col-md-6">
+				<label for="id-number">Student ID</label>
+				<input type="text" class="form-control" name="id_number" value="<%=student.id_number%>" readonly required>
```

Comment:
I don't think we need to show the Student ID field when someone is adding a student for the first time - it should just be for when they are editing/viewing